packages/reports/src/Components/Elements/Modals/AccountVerificationPendingModal/account-verification-pending-modal.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Button, Modal } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+
+type TAccountVerificationPendingModalProps = {
+    is_visible: boolean;
+    onConfirm: () => void;
+};
+
+const AccountVerificationPendingModal = ({ is_visible, onConfirm }: TAccountVerificationPendingModalProps) => (
+    <Modal
+        small
+        is_open={is_visible}
+        title={localize('Pending verification')}
```

Comment:
Should we use Localize Component here ?

Reply:
I copied this component from dtrader package, let's not do any refactoring here to avoid differences between reports and dtrader packages.

Reply:
Localize component will not make any difference here. :)
Especially, if this component is placed in components package and reused

Reply:
@maryia-deriv we have a lot of component duplications among the packages, in fact, the whole reports package is just a duplication of dtrader, so I honestly don't see why we should refactor it here or have nitpicks over the code which we are okay with in other packages) I think if we are not okay with these sort of things, we should refactor the whole package and components separately. This PR is about fixing the issue, so I fixed it in an exact way how it's done in dtrader, if there was a requirement to refactor, I'd move it to components as well but this needs proper work to do it for other components as well )

Reply:
@maryia-deriv I changed to Localize here, but I'm not gonna refactor the whole thing. The component is tightly coupled with trade-store and it's not worth effort spending time on making lots of changes here.